mono/utils/mono-state.h

```diff
@@ -24,14 +24,16 @@ typedef enum {
 	MonoSummaryNone = 0,
```

Comment:
You should just remove the values, the increment will be done automatically by the compiler.

Reply:
Because the managed API consumers are only getting an integer, I wanted to try to make the header as documenting as possible about what a failure level means. 

I should probably just add comments/constants on the managed side and remove them from here. This is corlib, so it'll be kept in sync.